category.php

```diff
@@ -40,6 +40,11 @@
     $content = do_blocks($template);
     $context['listing_page_content'] = $content;
     $context['page_category'] = 'Listing Page';
+    $news_page = (int) get_option('page_for_posts');
```

Comment:
What do you think if we move all this repeated code within the [`Context` `Class`](https://github.com/greenpeace/planet4-master-theme/blob/42068a4ea3bf73339e78d20677545cdf47c7e9d4/src/Context.php)?

Reply:
I 100% agree! PR updated 🙂